source/_includes/site/sidebar.html

```diff
@@ -48,11 +48,37 @@ <h1 class="title delta">On this page</h1>
     {%- endunless -%}
   {% endif %} 
 
+  {%- if page.ha_platforms.first -%}
+  <section class="aside-module grid__item one-whole lap-one-half">
+    <div class='section'>
+      <h1 class="title delta">Platforms</h1>
+      This integration creates entities from the following platforms.
+      See these general pages for details on using their triggers and actions in automations:
+    </div>
+    <div class='section'>
+      <ul class='unstyled'>
+      {%- for platform in page.ha_platforms -%}
+        <li>
+          <a href='/integrations/{{ platform }}'>{{ platform | capitalize }}</a>
+        </li>
+      {%- endfor -%}
+      </ul>
+    </div>
+  </section>
+  {%- endif -%}
+
   {%- if page.ha_category.first -%}
     <section class="aside-module grid__item one-whole lap-one-half">
       <div class='section'>
         <h1 class="title delta">Categories</h1>
-        <ul class='divided'>
+        {% if page.ha_integration_type == "integration" %}
+          Find other integrations in these categories:
+        {% else %}
+          Search for integrations built from this:
+        {% endif %}
+      </div>
+      <div class='section'>
+        <ul class='unstyled'>
```

Comment:
not sure if this is correct for categories. Categories are only used to group documentation. They might use the same term as a building block, but it's not the same. I would not change this section for now. Let me get another opinion.

Reply:
I considered that too - my idea in the end was that each of the building block pages is in a category of its own name, even if not every category has a building block. Up to you guys though.

Reply:
I considered that too - my idea in the end was that each of the building block pages is in a category of its own name, even if not every category has a building block. Up to you guys though.

Reply:
>  I would not change this section for now. Let me get another opinion.

I agree with that. Let's adjust it.